### PR TITLE
TODO Items

### PR DESCRIPTION
**Pull request by kennedyengineering**
Made several quality of life improvements.

TODO should be completed before merge

### CODE REVIEW

File: stable-baselines3/sb3.py [hunk 1]
```diff
@@ -34,14 +34,17 @@ def __init__(self, check_freq: int, log_dir: str, verbose: int = 1):
         super(SaveOnBestTrainingRewardCallback, self).__init__(verbose)
         self.check_freq = check_freq
         self.log_dir = log_dir
-        self.save_path = os.path.join(log_dir, "best_model")
```
Comment (luisdavidgarcia) [inline stable-baselines3/sb3.py:37]: looks good nice touch with the log dot to avoid conflicting names

File: stable-baselines3/sb3.py [hunk 2]
```diff
@@ -224,6 +225,12 @@ def parse_args(input=sys.argv[1:]):
             help="Enable learning rate scheduler",
             action="store_true",
         )
+        parser.add_argument(
```
Comment (luisdavidgarcia) [inline stable-baselines3/sb3.py:250]: Nice touch don't want to be slowed down by frequent updates

**Review by luisdavidgarcia — APPROVED** [review]
LGTM

**Comment by kennedyengineering** [comment]
Made more changes. Uses evaluation environment to determine quality of agent. Should be tested, might be bugs.

Also, more TODO's added

**Comment by kennedyengineering** [comment]
Add case where n_updates == 0, shouldn't pass if statement?

**Pull request merged**